src/routes/user/index.js

```diff
@@ -19,17 +20,21 @@ export async function userRoutes(server) {
       const client = await server.pg.pgWriter.connect();
       const { username, password, email } = req.body;
       
+      // hash password using bcrypt
+      const hashedPassword = await hashPassword(password);
```

Comment:
actually when we name function names that actually create something can we say "create[what_ou're creating]" so in this case "createHashedPassword".